wbooks-api/spec/factories/book.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :book do
+    factory :valid_book do
```

Comment:
Instead of doing this i prefer using traits or setting a field value in the test